spec/unit/blockscore/question_set_spec.rb

```diff
@@ -54,17 +54,25 @@ module BlockScore
       let(:incorrect_answers) { QuestionSetHelper.incorrect_answers(question_set.questions) }
 
       context 'correct answers' do
-        before { question_set.score(correct_answers) }
-
-        its(:score) { should eq 100.0 }
-        it { expect(question_set.score(correct_answers)).to eq 100.0 }
+        it { expect(question_set.score(correct_answers)).to equal 100.0 }
       end
 
       context 'incorrect answers' do
-        before { question_set.score(incorrect_answers) }
+        it { expect(question_set.score(incorrect_answers)).to equal 0.0 }
+      end
+
+      context 'malformed answers' do
+        it { expect { question_set.score([]) }.to raise_error InvalidRequestError }
```

Comment:
Is there a specific error message you can match on, or do all `InvalidRequestError` instances use the same error message? If the error message can differ then I would typically assert it in the raise_error call.